OpenOversight/tests/routes/test_other.py

```diff
@@ -36,3 +36,18 @@ def test_user_can_access_profile(mockdata, client, session):
         assert "User Email" not in rv.data.decode("utf-8")
         # Toggle button should not appear for this non-admin user
         assert "Edit User" not in rv.data.decode("utf-8")
+
+
+def test_user_can_access_profile_differently_cased(mockdata, client, session):
+    with current_app.test_request_context():
+        login_user(client)
+
+        rv = client.get(
+            url_for("main.profile", username="TEST_USER"), follow_redirects=True
+        )
+        assert "test_user" in rv.data.decode("utf-8")
```

Comment:
I'd make the `utf-8` a constant.

Reply:
It's a close call for me. I think it's more trouble than worth since we probably would want to change it everywhere then